server/constants/constants.go

```diff
@@ -0,0 +1,46 @@
+package constants
```

Comment:
From https://go.dev/blog/package-names

> Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains.


What value does a separate package provide?

Reply:
@hanzei Keeping things in separate package helps us in writing unit tests and creating mocks for packages.

Reply:
4/5 that a `constants` package is not an idiomatic way to store variables, as it contradicts the above quote.